src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -296,6 +298,17 @@ class FrontendGenImpl {
 
     // TODO: Handle optional inputs.
     auto op = builder_.create<T>(UnknownLoc(), outputTypes, inputs, attributes);
+
+    // Type inference for results.
+    if (mlir::ResultTypeInferenceInterface opWithTypeInference =
```

Comment:
Using `auto` here would be more appropriate, since the type is specified right after this line, there's no need to repeat.